csharp-basic-topics/RemoveIllegalCharsFromPathAndFilenames/RemoveIllegalCharsFromPathAndFilenames/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿HashSet<char> invalidPathChars = new HashSet<char>(Path.GetInvalidPathChars());
+HashSet<char> invalidFilenameChars = new HashSet<char>(Path.GetInvalidFileNameChars());
+
+List<string> pathList = new List<string>
+{
+    "A://ValidPath/SomeFolder/Invalid??File??Broken.nah",
+    "B:\\InValid:|:Path/",
+    "C://IsThisavalidpath??\0/",
+    "D://Thi*IsIN\nv00ALID/<</",
+    "E://Users/ValidPath/docs/Storyboard_2023.docx",
+    "F://Users/Photos/Cats/FluffyCat.png",
+    "G://Inv@7**\\*lidPath/",
+    "H://Test/This<IS>Ano<>therInvalidPath\r/",
+    "I://ProgramFiles/SomeApplication/:Invalid:Filename.txt"
+};
+
+// Checking for invalid paths
+Console.WriteLine("Conventional Logic Result:");
+foreach (var str in StringMethods.CheckForInvalid(pathList, invalidPathChars)) { Console.WriteLine(str); }
+
+Console.WriteLine("\nLINQ Result:");
```

Comment:
I would rather do another `Console.WriteLine()` instead of using `\n`. Instead of using \n correct would be to use `Environment.NewLine` if you really want to use it.